OJP/OJP_FareSupport.xsd

```diff
@@ -217,9 +230,20 @@
 		<xs:annotation>
 			<xs:documentation>Reference to a a precondition to access a service or to purchase a FARE PRODUCT issued by an organisation that may not be a PT operator (eg: military card, concessionary card, etc).</xs:documentation>
 		</xs:annotation>
-		<xs:simpleContent>
-			<xs:extension base="EntitlementProductCodeType"/>
-		</xs:simpleContent>
+		<xs:sequence>
```

Comment:
I am wondering whether with this change the name `EntitlementProductRefStructure` is still valid or if it should be called `EntitlementProductStructure` now (without the `Ref`) as it contains several elements itself and no longer is a sole reference.

Reply:
Changed that as well